src/Panel/Ui/FilePreviews/FileDefaultPreview.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Kirby\Panel\Ui\FilePreviews;
+
+use Kirby\Cms\File;
+use Kirby\Panel\Ui\FilePreview;
+
+/**
+ * Fallback file preview component
+ *
+ * @package   Kirby Panel
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ * @since     5.0.0
+ * @internal
+ */
+class FileDefaultPreview extends FilePreview
+{
+	public string $component = 'k-file-default-preview';
+
+	/**
+	 * Accepts any file as last resort
+	 */
+	public static function accepts(File $file): bool
+	{
+		return true;
+	}
+
+	/**
+	 * Icon or image to display as thumbnail
+	 */
+	public function image(): array|null
+	{
+		return $this->file->panel()->image([
+			'back'  => 'transparent',
+			'ratio' => '1/1'
+		], 'cards');
+	}
```

Comment:
`Question`: or should this be in the abstract class for all previews?

Reply:
It looks to me like it could also be in the abstract class.